crates/release-automation/src/tests/cli.rs

```diff
@@ -666,3 +673,58 @@ fn multiple_subsequent_releases() {
         std::thread::sleep(std::time::Duration::new(1, 0));
     }
 }
+
+#[test]
+fn apply_dev_versions_works() {
+    // todo: test this or remove the command altogether
```

Comment:
todo or not todo

Reply:
my current thinking is that we'll need the command but i haven't fully dug into the details of how `cargo publish/check` uses crates.io vs local path for dependencies who are available at both.

Reply:
i added a test for this :+1: